src/mobile/src/ui/components/CustomTextInput.js

```diff
@@ -537,6 +537,7 @@ class CustomTextInput extends Component {
                         onChangeText={(text) => this.onChangeText(text)}
                         selectionColor={theme.input.alt}
                         underlineColorAndroid="transparent"
+                        autoCorrect={false}
```

Comment:
Won't this affect all text fields, even ones where users would expect auto-correct (e.g. `message` field when sending a transaction)?

Reply:
Good catch. This was a test and wasn’t meant to be committed.